lib/assets/data/fee_types.csv

```diff
@@ -1,100 +1,101 @@
+70,Trial not proceed uplift,TNU,MITNU,,TRUE,Fee::MiscFeeType,agfs;agfs_scheme_9;agfs_scheme_10,,FALSE,
+71,Unsuccessful application to vacate a guilty plea (half day),PAH,MIUAV1,,TRUE,Fee::MiscFeeType,agfs;agfs_scheme_9;agfs_scheme_10,,FALSE,
+72,Unsuccessful application to vacate a guilty plea (whole day),PAW,MIUAV2,,TRUE,Fee::MiscFeeType,agfs;agfs_scheme_9;agfs_scheme_10,,FALSE,
+73,Unsuccessful application to vacate a guilty plea (half day uplift),PHU,MIUAV3,,TRUE,Fee::MiscFeeType,agfs;agfs_scheme_9;agfs_scheme_10,,FALSE,
+74,Unsuccessful application to vacate a guilty plea (whole day uplift),PWU,MIUAV4,,TRUE,Fee::MiscFeeType,agfs;agfs_scheme_9;agfs_scheme_10,,FALSE,
+75,Written / oral advice,WOA,MIWOA,,TRUE,Fee::MiscFeeType,agfs;agfs_scheme_9;agfs_scheme_10,,TRUE,
+76,Wasted preparation fee,WPF,MIWPF,,TRUE,Fee::MiscFeeType,agfs;agfs_scheme_9;agfs_scheme_10,,TRUE,
+77,Trial,GTRL,GRTRL,9999,FALSE,Fee::GraduatedFeeType,lgfs,,FALSE,
+78,Retrial,GRTR,GRRTR,9999,FALSE,Fee::GraduatedFeeType,lgfs,,FALSE,
+79,Guilty plea,GGLTY,GRGLT,9999,FALSE,Fee::GraduatedFeeType,lgfs,,FALSE,
+80,Discontinuance,GDIS,GRDIS,9999,FALSE,Fee::GraduatedFeeType,lgfs,,FALSE,
+81,Cracked trial,GCRAK,GRRAK,9999,FALSE,Fee::GraduatedFeeType,lgfs,,FALSE,
+82,Cracked before retrial,GCBR,GRCBR,9999,FALSE,Fee::GraduatedFeeType,lgfs,,FALSE,
+83,Hearing subsequent to sentence,XH2S,FXH2S,,FALSE,Fee::FixedFeeType,lgfs,,FALSE,
+84,Vary/discharge an ASBO s1c Crime and Disorder Act 1998,XASB,FXASB,,FALSE,Fee::FixedFeeType,lgfs,83,FALSE,
+85,Alteration of Crown Court sentence s155 Powers of Criminal Courts (Sentencing Act 2000),XALT,FXALT,,FALSE,Fee::FixedFeeType,lgfs,83,FALSE,
+86,Assistance by defendant: review of sentence s74 Serious Organised Crime and Police Act 2005,XASS,FXASS,,FALSE,Fee::FixedFeeType,lgfs,83,FALSE,
+87,Evidence provision fee,XEVI,MIEVI,,FALSE,Fee::MiscFeeType,lgfs,,FALSE,
+88,Costs judge application,XCJA,MICJA,,FALSE,Fee::MiscFeeType,lgfs,,FALSE,
+89,Costs judge preparation,XCJP,MICJP,,FALSE,Fee::MiscFeeType,lgfs,,FALSE,
+90,Case uplift,XUPL,MIUPL,,FALSE,Fee::MiscFeeType,lgfs,,FALSE,
+91,Warrant Fee,XWAR,WARR,,FALSE,Fee::WarrantFeeType,lgfs,,FALSE,
+92,Contempt,ZCON,FXCON,,TRUE,Fee::FixedFeeType,agfs,,FALSE,
+93,Effective PCMH,IPCMH,INPCM,,FALSE,Fee::InterimFeeType,lgfs,,FALSE,
+94,Trial start,ITST,INTDT,,FALSE,Fee::InterimFeeType,lgfs,,FALSE,
+95,Retrial New solicitor,IRNS,INRNS,,FALSE,Fee::InterimFeeType,lgfs,,FALSE,
+96,Retrial start,IRST,INRST,,FALSE,Fee::InterimFeeType,lgfs,,FALSE,
+97,Disbursement only,IDISO,INDIS,,FALSE,Fee::InterimFeeType,lgfs,,FALSE,
+98,Warrant,IWARR,INWAR,,FALSE,Fee::InterimFeeType,lgfs,,FALSE,
+99,Transfer,TRANS,TRANS,,FALSE,Fee::TransferFeeType,lgfs,,FALSE,
+100,Further case management hearing,FCM,MIFCM,,TRUE,Fee::MiscFeeType,agfs;agfs_scheme_10,,FALSE,
+101,Ground rules hearing,GRH,MIGRH,,TRUE,Fee::MiscFeeType,agfs;agfs_scheme_10,,FALSE,
+102,Daily attendance fee (2+),DAT,BADAT,999,TRUE,Fee::BasicFeeType,agfs;agfs_scheme_10,,FALSE,2
```

Comment:
New **fee type** under for scheme 10: "Daily attendance fee (2+)", that replaces all the existent "Daily attendance fee X" existent in scheme 9.